app/kumactl/cmd/generate/generate_dataplane_token.go

```diff
@@ -13,7 +13,8 @@ type generateDataplaneTokenContext struct {
 	*kumactl_cmd.RootContext
 
 	args struct {
-		dataplane string
+		dataplane string // Deprecated: remove in next major version of Kuma 1.1
```

Comment:
We have to find a way to track these deprecations. Either we have a fixed tag, like `Deprecated:` or we track in a dedicated doc.